Examples/Python/src/ExaTrkXTrackFinding.cpp

```diff
@@ -140,24 +139,6 @@ void addExaTrkXTrackFinding(Context &ctx) {
 #endif
 
 #ifdef ACTS_EXATRKX_ONNX_BACKEND
-  {
-    using Alg = Acts::OnnxMetricLearning;
-    using Config = Alg::Config;
-
-    auto alg =
-        py::class_<Alg, Acts::GraphConstructionBase, std::shared_ptr<Alg>>(
-            mex, "OnnxMetricLearning")
-            .def(py::init([](const Config &c, Logging::Level lvl) {
-                   return std::make_shared<Alg>(
-                       c, getDefaultLogger("MetricLearning", lvl));
-                 }),
-                 py::arg("config"), py::arg("level"))
-            .def_property_readonly("config", &Alg::config);
-
-    auto c = py::class_<Config>(alg, "Config").def(py::init<>());
-    ACTS_PYTHON_STRUCT(c, modelPath, spacepointFeatures, embeddingDim, rVal,
-                       knnVal);
-  }
```

Comment:
This goes away completely?

Reply:
Yes, at least for now. Maybe it can be added later again, but I don't have capacities to update it.
I you think its better we can also keep the code around though...